vars/capsuleSystemTests.groovy

```diff
@@ -10,6 +10,18 @@ void buildGoBinary(String directory, String outputBinary, String packages) {
   }
 }
 
+def boxPublicIP() {
+    def boxIp = "unknown";
+    try {
+        sh script: 'curl ifconfig.co',returnStdout:true
```

Comment:
you need to assign this to `boxIp` var

Reply:
and possibly add `.trim()` to remove newline

Reply:
done